common.py

```diff
@@ -8,8 +8,7 @@
 import subprocess as sp
 import re
 import logging
-
-log = logging.getLogger(__name__)
+log = logging.getLogger()
```

Comment:
I suppose we will talk more about how we do logging.

I am in favor of identifying the module and function we are logging from....

I guess more to come.